test/manage/components/app-test.js

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+require("babel-polyfill");
+
+import chai, { expect } from "chai";
```

Comment:
When last we talked about testing, we agreed on `assert` not `expect`.

It's not really a problem, but it's a surprise that would have been nice to get at least a notice about.

Note `lockbox-datastore` is using `assert`.

Reply:
I think I just used `expect` because it's what @m8ttyB's original patch started with (and I forgot what we'd agreed on). However, `expect` *does* give us more options in terms of plugins so maybe it's the better choice long-term...